gitlab.go

```diff
@@ -190,6 +191,7 @@ func newClient(httpClient *http.Client, tokenType tokenType, token string) *Clie
 	c.Settings = &SettingsService{client: c}
 	c.SystemHooks = &SystemHooksService{client: c}
 	c.Users = &UsersService{client: c}
+	c.Builds = &BuildsService{client: c}
```

Comment:
_nitting_ Could you please insert the new `c.Builds` service following alphabetic ordering?